app/v1/services/DeleteCodingOutService.scala

```diff
@@ -28,25 +29,35 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class DeleteCodingOutService @Inject() (connector: DeleteCodingOutConnector) extends BaseService {
+class DeleteCodingOutService @Inject()(connector: DeleteCodingOutConnector) extends BaseService {
 
   def deleteCodingOut(request: DeleteCodingOutParsedRequest)(implicit
-      ctx: RequestContext,
-      ec: ExecutionContext): Future[Either[ErrorWrapper, ResponseWrapper[Unit]]] = {
+                                                             ctx: RequestContext,
+                                                             ec: ExecutionContext): Future[Either[ErrorWrapper, ResponseWrapper[Unit]]] = {
 
-    connector
-      .deleteCodingOut(request)
-      .map(_.leftMap(mapDownstreamErrors(errorMap)))
+    val result = EitherT(connector.deleteCodingOut(request)).leftMap(mapDownstreamErrors(errorMap))
```

Comment:
Can further simplify this:
`
connector.deleteCodingOut(request).map(_.leftMap(mapDownstreamErrors(errorMap)))
`

Can then remove line 40